crates/c_api/src/types/extern.rs

```diff
@@ -75,6 +76,10 @@ pub extern "C" fn wasm_externtype_kind(et: &wasm_externtype_t) -> wasm_externkin
 
 /// Returns a mutable reference to the [`wasm_externtype_t`] as [`wasm_functype_t`].
 #[no_mangle]
+#[cfg_attr(
+    feature = "prefix-symbols",
+    export_name = "wasmi_wasm_externtype_as_functype"
+)]
```

Comment:
This should use the `#[cfg_attr(feature = "prefix-symbols", wasmi_c_api_macros::prefix_symbol)]` attribute. Probably an oversight due to text replacement without multi-lines.

Reply:
Also many more occurrences in the same file.